collection-scripts/common.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+export BASE_COLLECTION_PATH="${BASE_COLLECTION_PATH:-/must-gather}"
+declare -a DEFAULT_NAMESPACES=("openstack" "openstack-operators")
+export DEFAULT_NAMESPACES
+NAMESPACE_PATH=${BASE_COLLECTION_PATH}/namespaces
+export NAMESPACE_PATH
+
+# k8s services that must be gather from the openstack
+# ctlplane namespace
+declare resources=(
+    "services"
+    "routes"
+    "daemonset"
+    "replicaset"
+    "deployments"
+    "statefulset"
+    "jobs"
+)
```

Comment:
Just wondering do we want to capture any information about webhooks?

In OCP must gather we do capture some information about webhooks[1]

[1] https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/40096/rehearse-40096-pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl/1674096584003948544/artifacts/openstack-operator-build-deploy-kuttl/openstack-k8s-operators-gather/artifacts/must-gather/quay-io-openshift-release-dev-ocp-v4-0-art-dev-sha256-0da5ac4ccc0dd27b39d98564c43753501630de95e8ff3b173a8263df4ef651cd/web_hooks/

Reply:
Yes, adding it in my next iteration

Reply:
we might need `buildconfig` related resources in that list (which is relevant for CI)

Reply:
Done.